pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/auth_proxy_role.go

```diff
@@ -43,6 +44,11 @@ func (f *AuthProxyRole) SetTemplateDefaults() error {
 const proxyRoleTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: proxy-role
+    app.kubernetes.io/component: auth-proxy
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

The component here is a permission / rbac that can be applied for the auth proxy container

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.